src/pages/docs/guides/chat/build-embedded-chat.mdx

```diff
@@ -0,0 +1,414 @@
+  - For games, use one room per game session for games such as chess, or for a MMORPG, allocate a room for each location or instance of a quest.
+  - For deliveries, appointments, or calls, there is a clear 1:1 mapping of task to room and users join the room when they join the task.
+- Will users be active at the same time, or will they come and go with periods of inactivity?
+  - Only attaching to rooms when they are needed is more cost-effective and saves resources.
+  - For example, during a delivery, the chat room doesn't always have to be active. With a good strategy for push notifications to alert inactive users of new messages, they can attach for very short periods of time for each delivery.
+- Do you need any spam prevention measures applied before allowing users to join rooms?
+  - For example, in a support chat scenario, use a captcha or other measures before allowing users to join a room or create a new ticket.
+- Do you need long-term archiving of the chat history?
+  - Check out the guide for [exporting chat data](/docs/guides/chat/export-chat) for an overview of your options.
+
+Ably has been tested to handle millions of concurrent connections and rooms. Whether you're managing 100 support conversations or 100,000 simultaneous delivery chats, the architecture remains the same - no re-engineering required as you grow.
+
+## Authentication
+
+Authentication is a design decision. 
+
+In some cases, you'll want to tie participants to their identity in your application. In others, anyone can join and participate. Ably Chat supports both of these scenarios - you don't need to go through the hassle of setting up users in Ably and trying to keep them in sync with your application. Every client has a `clientId`. If you want to allow random users, just use a random string. If you want to tie it to your application, then use the ID of the user or some other well-known identifier. Just make sure you can tell them apart if you allow both at the same time, for example by using distinct prefixes.
+
+<Code>
+```javascript
+const jwt = require("jsonwebtoken");
+
+const header = {
+    "typ": "JWT",
+    "alg": "HS256",
+    "kid": "{{ API_KEY_NAME }}"
+}
+const currentTime = Math.round(Date.now() / 1000);
+
+// The clientId could be the users ID in your application,
+// or a random string if you want to make users anonymous.
+//
+// The capabilities here allow the holder of this token to publish and subscribe to messages,
+// typing indicators and room reactions in the chat room called "foo".
+
+const claims = {
+    "iat": currentTime, /* current time in seconds */
+    "exp": currentTime + 14400, /* time of expiration in seconds */
+    "x-ably-capability": "{\"foo\":[\"publish\", \"subscribe\"]}",
+    "x-ably-clientId": "your-client-id",
```

Comment:
Should we be showing an example of a prefix + randomisation as well as an ID from another system as per the text?

Reply:
not sure if it adds a lot of value? It'll need a new code block, maybe too much?